pkg/kubeapiserver/options/storage_versions.go

```diff
@@ -17,14 +17,12 @@ limitations under the License.
 package options
 
 import (
+	"sort"
 	"strings"
 
+	"github.com/spf13/pflag"
```

Comment:
tho these changes might not be necessary. 😃do you think we should reformat "import" declarations here? or it looks too "sparse".

Reply:
squashing this in is fine.